LICENCE-THIRD-PARTY

```diff
@@ -41,6 +159,510 @@ licensed under MIT licence.
 third-party-licences/Detectorjs.txt
 ==============================================================================
 
+==============================================================================
+Django
+------------------------------------------------------------------------------
+https://github.com/django/django
+Copyright Django Software Foundation and individual contributors.
+    Includes code from the Python standard library
+    which is licenced under the Python licence
+licensed under the three-clause BSD licence.
+third-party-licences/django.txt
+==============================================================================
+
+==============================================================================
+django-bidi-utils
+------------------------------------------------------------------------------
+https://github.com/MeirKriheli/django-bidi-utils
+Copyright 2009 Meir Kriheli
+licensed under MIT licence.
+third-party-licences/django-bidi-utils.txt
+==============================================================================
+
+==============================================================================
+Django Debug Toolbar
+------------------------------------------------------------------------------
+https://github.com/jazzband/django-debug-toolbar
+Copyright Rob Hudson and individual contributors
+licensed under BSD licence.
+third-party-licences/django-debug-toolbar.txt
+==============================================================================
+
+==============================================================================
+django-haystack (AKA Haystack)
+------------------------------------------------------------------------------
+http://haystacksearch.org/
+Copyright 2009-2011 Daniel Lindsley
+licensed under BSD licence.
+No known licence file to include
```

Comment:
Is this relevant? https://github.com/django-haystack/django-haystack/blob/master/LICENSE

Reply:
Yes, how did I miss that??